javascript/javascript3/week1/weatherApp/script.js

```diff
@@ -0,0 +1,109 @@
+// @ts-nocheck
+window.addEventListener('load', () => {
+  let lat;
+  let long;
+  let geoBtn = document.querySelector('.geoBtn');
```

Comment:
For the naming of HTML classes, I would use Kebab case (kebab-case), and reserve the camelCase for js variable names.